From 7e9fa50ab3734b63dd62b381e0f17bf6abdd8c40 Mon Sep 17 00:00:00 2001
From: Jianchuan Wang <jianchuan.wang@windriver.com>
Date: Fri, 12 Jun 2015 10:05:48 +0800
Subject: [PATCH] Allow multiple attempts to connect to relayd

It is unclear why a session needs to be made
unusable after a failure to connect to relayd
since a check for a relayd connection is
always made before a session can be configured.

The behaviour was introduced in:
d9078d0c000d04d49c599a72c1a725026b636ec0

Signed-off-by: Mikael Beckius <mikael.beckius@windriver.com>
---
 src/bin/lttng-sessiond/cmd.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/src/bin/lttng-sessiond/cmd.c b/src/bin/lttng-sessiond/cmd.c
index 044e9ee..53b2bb0 100644
--- a/src/bin/lttng-sessiond/cmd.c
+++ b/src/bin/lttng-sessiond/cmd.c
@@ -657,14 +657,6 @@ close_sock:
 	free(rsock);
 
 error:
-	if (ret != LTTNG_OK) {
-		/*
-		 * The consumer output for this session should not be used anymore
-		 * since the relayd connection failed thus making any tracing or/and
-		 * streaming not usable.
-		 */
-		consumer->enabled = 0;
-	}
 	return ret;
 }
 
-- 
1.9.1

